### DISCUSSION TITLE
get style attribute from components

### DISCUSSION THREAD

**Discussion by audn** [remarkjs/remark#1060] [General]
Hi,
Is it possible to get the `style` attribute from an `<img>` element?

In the readme it says..
> The props that are passed are what you probably would expect: an a (link) will get href (and title) props, and img (image) an src, alt and title, etc. There are some extra props passed.

But I'd like to retrieve the `style` attribute as well, if possible.

This code:
```html
<img src="img" alt="hey" title="ayo" style="border-radius: 0.4rem; width: 30rem" />
```
Returns:
```js
{src: 'img', alt: 'hey', title: 'ayo'}
```

Any ideas? Thanks!

**Comment by wooorm** [comment]
Where? This organisation is giant. Can you please explain your question in detail? See https://github.com/remarkjs/.github/blob/main/support.md#asking-quality-questions.
